mmcls/models/tta/score_tta.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import List
+
+from mmengine.model import BaseTTAModel
+from mmengine.structures import BaseDataElement
+
+from mmcls.registry import MODELS
+from mmcls.structures import ClsDataSample
+
+
+@MODELS.register_module()
+class AverageScoreTTAModel(BaseTTAModel):
+
+    def merge_preds(
+        self,
+        data_samples_list: List[List[ClsDataSample]],
+    ) -> List[BaseDataElement]:
+        merged_data_samples = []
+        for data_samples in data_samples_list:
+            merged_data_samples.append(self.merge_single_sample(data_samples))
+        return merged_data_samples
+
+    def merge_single_sample(self, data_samples):
```

Comment:
If here is a public function, need docstring and name `merge_single_sample` may make people confused if the params are the name of `data_samples`

Reply:
I have changed it to private method.